src/Facebook/FacebookClient.php

```diff
@@ -197,7 +197,7 @@ public function prepareRequestMessage(FacebookRequest $request)
      */
     public function sendRequest(FacebookRequest $request)
     {
-        if (get_class($request) === 'FacebookRequest') {
+        if (get_class($request) === 'Facebook\FacebookRequest') {
```

Comment:
There isn't an interface but `FacebookBatchRequest` extends from `FacebookRequest`. When you send a batch request it [eventually calls the `sendRequest()` method](https://github.com/facebook/facebook-php-sdk-v4/blob/master/src/Facebook/FacebookClient.php#L243-L246).